projects/experimental/components/input-phone-international/input-phone-international.template.html

```diff
@@ -0,0 +1,65 @@
+<ng-container *tuiTextfieldContent>
+    <button
+        appearance="textfield"
+        tabindex="-1"
+        tuiButton
+        tuiChevron
+        type="button"
+        class="t-ipi-select"
+        [attr.data-mode]="mode()"
+        [class.t-ipi-select_readonly]="readOnly()"
+        [disabled]="disabled()"
+        [size]="size()"
+        [tuiAppearanceFocus]="open()"
+        (mousedown.prevent)="interactive() && open.set(!open())"
+    >
+        <img
+            class="t-ipi-flag"
+            [alt]="names()?.[code()]"
+            [class.t-ipi-flag_small]="size() === 's'"
+            [src]="code() | tuiFlag"
+        />
+    </button>
+</ng-container>
+<ng-container *tuiTextfieldDropdown>
+    <div
+        *ngIf="countrySearch"
+        #element
+        class="t-ipi-search"
```

Comment:
Should we hide this gap ?

<img width="454" alt="gap-demo" src="https://github.com/user-attachments/assets/e749b5b6-31a9-46b3-ab54-7314b0860ed7" />


https://github.com/user-attachments/assets/f80a3c63-300f-4fb8-96bd-cec8af3dacf0

Reply:
I faced this problem only in Chrome DevTools (Mobile emulation).
**Not** reproducible for IOS safari

Reply:
Pretty sure it's subpixel gap in non-round scale in emulator.